EIPS/eip-1193.md

```diff
@@ -200,7 +203,10 @@ ethereum
 
 // Example 2: Log last block
 ethereum
-  .request('eth_getBlockByNumber', ['latest', 'true'])
+  .request({
+    method: 'eth_getBlockByNumber',
+    params: ['latest', 'true'],
```

Comment:
This line can also be written like this, right?

```js
params: {
  latest: true
}
```

Reply:
This example seems a little more straight-forward to understand when written as an object instead of an array.

Reply:
Hm, I don't know if the params can be given in object form. I imagine it's just following: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getblockbynumber

Reply:
By the method's interface the params can certainly be given as an object, but whatever the specific RPC method's definition says, goes.

Reply:
Ah nevermind, I confused myself. However the type annotation (on line 32) here makes it seems like params can either be an array of anything or an object?

```ts
params?: Array<any> | { [key: string]: any };
```

Reply:
But in the official JSON-RPC, it seems like it is always an array.